curvefs/src/metaserver/partition.cpp

```diff
@@ -61,29 +75,42 @@ Partition::Partition(const PartitionInfo& paritionInfo,
 }
 
 // dentry
-MetaStatusCode Partition::CreateDentry(const Dentry& dentry, bool isLoadding) {
+MetaStatusCode Partition::CreateDentry(const Dentry& dentry) {
```

Comment:
give more meaningful code comments or delete it.

Reply:
> give more meaningful code comments or delete it.

fixed.